build/build-push-bundles.sh

```diff
@@ -1,27 +1,49 @@
 #!/bin/bash
 
-set -e
+set -ex
+
+CONTAINER_TOOL="${CONTAINER_TOOL:-podman}"
 
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." >/dev/null 2>&1 && pwd)"
 
+if [[ "${CONTAINER_TOOL}" == "podman" ]]; then
+    host_platform="$(podman info --format '{{.Version.OsArch}}')"
+else
+    host_platform="$(docker info --format '{{.ClientInfo.Os}}/{{.ClientInfo.Arch}}')"
+fi
+
 for dirpath in $(find ${BASE_DIR}/operators -type d -print -mindepth 1 -maxdepth 1)
 do
     cd "$dirpath"
     operator_name=$(basename ${dirpath})
     echo "* Handling operator ${operator_name} ..."
     
-    podman manifest rm "${operator_name}" || true
-    podman manifest create "${operator_name}"
     build_img="quay.io/stolostron-grc/${operator_name}:latest"
     for arch in amd64 arm64 s390x ppc64le
     do
-        podman build -f Dockerfile --manifest "${operator_name}" \
-          --platform "linux/${arch}" --build-arg "TARGETARCH=${arch}"
+        echo "::group::Building ${operator_name} for ${arch}"
+        ${CONTAINER_TOOL} build -f Dockerfile -t "${build_img}-${arch}" \
+          --platform "linux/${arch}" --build-arg "TARGETARCH=${arch}" \
+          --build-arg "HOSTPLATFORM=${host_platform}" .
+        ${CONTAINER_TOOL} push "${build_img}-${arch}"
```

Comment:
Why is each image getting pushed directly rather than only including it in the manifest?

Reply:
Because `docker` only generates some of the manifest metadata when the image gets pushed to a registry :disappointed: if these aren't pushed, the `manifest create` will fail. I suspect this is why some image repositories have a bunch of flotsam: https://quay.io/repository/operator-framework/opm?tab=tags&tag=latest